src/components/label-pill/LabelPill.scss

```diff
@@ -0,0 +1,56 @@
+@import '../../styles/variables';
+
+.bdl-LabelPill {
+    margin-right: 4px;
```

Comment:
To keep this flexible, we should not include margins for core components and allow the parent apps to designate spacing as appropriate. Expected usage of this could be something like:

`<LabelPill className="mrs" />`

This is pretty clear with our usage of `<Button>` where we were frequently forced to override the outer margin

Reply:
I agree, but if it is pretty clear, why was it added initially to these other components? has whatever existed then that made it a good idea changed? Perhaps the frequent overrides are a problem with the designs...that elements' associations vary more than they ought to. In past component libraries that I've built, the spacing between components was defined up front so everything fell into predicable units (and made overrides almost non-existent)

Also do we have a suite of mixins/classes for layout, or is "mrs" hypothetical?

Reply:
There was some talk a while back to use https://github.com/suitcss/suit/tree/master/packages/utils but unsure if it has the granularity of how we used to have "mrs".

If you think most parent apps will have to do 4px, maybe a SCSS variable can be provided here that they can use in their css.

Reply:
> Also do we have a suite of mixins/classes for layout, or is "mrs" hypothetical?

We do have utility classes: https://github.com/box/box-ui-elements/blob/master/src/styles/modifiers/_margins.scss

But we need to review the values and align with our grid size. After that I would say that is the primary unit of layout between elements, including add flex positioning helpers etc.